azurerm/resource_arm_key_vault_migration_test.go

```diff
@@ -340,6 +458,21 @@ func TestAzureRMKeyVaultMigrateState(t *testing.T) {
 				"access_policy.0.secret_permissions.5":       "recover",
 				"access_policy.0.secret_permissions.6":       "restore",
 				"access_policy.0.secret_permissions.7":       "set",
+				"access_policy.0.storage_permissions.#":      "14",
+				"access_policy.0.storage_permissions.0":      "backup",
+				"access_policy.0.storage_permissions.1":      "delete",
+				"access_policy.0.storage_permissions.2":      "deletesas",
+				"access_policy.0.storage_permissions.3":      "get",
+				"access_policy.0.storage_permissions.4":      "getsas",
+				"access_policy.0.storage_permissions.5":      "list",
+				"access_policy.0.storage_permissions.6":      "listsas",
+				"access_policy.0.storage_permissions.7":      "purge",
+				"access_policy.0.storage_permissions.8":      "recover",
+				"access_policy.0.storage_permissions.9":      "regeneratekey",
+				"access_policy.0.storage_permissions.10":     "restore",
+				"access_policy.0.storage_permissions.11":     "set",
+				"access_policy.0.storage_permissions.12":     "setsas",
+				"access_policy.0.storage_permissions.13":     "update",
```

Comment:
Did you want to remove the `v0_1_all` test case?  Or just the two storage ones that I had added?

Reply:
thinking about this some more, could we revert the changes in `resource_arm_key_vault_migration_test.go`? since there's no need to test for them in the state migration

Reply:
I attempted to revert the test file but tests began failing.  It appears that the migration tests, at minimum, all expect a terraform output of 
`access_policy.0.storage_permissions.#:0`.  Right now, the tests just include a single storage permission, and that storage permission does not change before or after the state migration (showing its not affected by it).  But I think we need it there due to how the tests need to read the policies from the resource.